ci-operator/step-registry/openshift-qe/orion/olmv1/openshift-qe-orion-olmv1-commands.sh

```
@@ -198,34 +198,40 @@ orion_exit_status=$?
-except IOError as e:
-    print(f"Error reading or writing file: {e}", file=sys.stderr)
-    sys.exit(1)
+    except ET.ParseError as e:
+        print(f"Error parsing XML file {file_path}: {e}", file=sys.stderr)
+        sys.exit(1)
+    except IOError as e:
+        print(f"Error reading or writing file {file_path}: {e}", file=sys.stderr)
+        sys.exit(1)
 EOF
 
   if [[ $? -ne 0 ]]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)